### ISSUE TITLE
release() is documented wrong

### ISSUE THREAD

**Issue by bmidgley** [iron-io/iron_mq_python#42]
README.md shows release has a signature:

queue.release(message_id, reservation_id, delay=30)

but the code shows it as

queue.release(message_id, delay=0, reservation_id=None)

I would submit a PR but I don't know if you think the docs are wrong or the code is wrong.

BTW the test calls it like the docs explain, so they should not work

**Comment by ulandj** [comment]
Hi @bmidgley,

I think you have the old version iron_mq_python. I looked the current code and code installed with pip, both look the same. And README.MD looks correctly.

##readme.md:
https://github.com/iron-io/iron_mq_python#release-a-reserved-message
```
queue.release(msg_id, delay=30)
```

##code:
https://github.com/iron-io/iron_mq_python/blob/master/iron_mq.py#L147
```python
def release(self, message_id, delay=0):
        url = "queues/%s/messages/%s/release" % (self.name, message_id)
        body = {}
        if delay > 0:
            body['delay'] = delay
        body = json.dumps(body)

        response = self.client.post(url, body=body,
                                    headers={"Content-Type":"application/json"})

        return response['body']
```

Thanks!

**Comment by treeder** [comment]
@ulandj he's referring to v3 which requires a reservation_id: https://github.com/iron-io/iron_mq_python/tree/v3#release-reserved-message

@bmidgley where do you see this code signature: 

```python
queue.release(message_id, delay=0, reservation_id=None)
```

The code and README show the same format from what I can see:

https://github.com/iron-io/iron_mq_python/blob/v3/iron_mq.py#L192

**Comment by edsrzf** [comment]
It changed in this commit: https://github.com/iron-io/iron_mq_python/commit/3bd6c8fd7bf399bf7ed2d183e4c1b7fc0a33bc69

That hasn't been released yet. The compatibility break is unfortunate, but I think the solution is just to release.

**Comment by treeder** [comment]
I see. Let's release then.

**Comment by edsrzf** [comment]
I've released version 0.6. Please update and call release as: `client.release(message_id, reservation_id, delay)`. (`delay` is optional and 0 by default).
